README.md.

```diff
@@ -1,4 +1,4 @@
-Rain water trapping problem
+# Rain water trapping problem

```

Comment:
Remove the last `.` in the file name.

~README.md.~
README.md

Reply:
It is not done.